handlers/new.go

```diff
@@ -24,14 +23,19 @@ const DefaultLeaderHost = "127.0.0.1:2379"
 var (
 	cfg           *client.Config
 	discHost      string
-	CurrentLeader lockstring.LockString
+	currentLeader lockstring.LockString
 	etcdMachines  []string
-	baseURI       = flag.String("host", "https://discovery.castle.io",
-		"base location for computed token URI")
+	baseURI       = flag.String("host", "https://discovery.castle.io", "base location for computed token URI")
+	etcdClient    client.KeysAPI
```

Comment:
instead of a package var, what if you follow the pattern of the Handler type in castled/pkg/api/handlers.go?